mmocr/utils/ocr.py

```diff
@@ -633,6 +749,10 @@ def det_recog_kie_inference(self, det_model, recog_model, kie_model=None):
 
     # Separate det/recog inference pipeline
     def single_inference(self, model, arrays, batch_mode, batch_size=0):
+        # todo: support batch mode for tesseract
+        if model == 'Tesseract':
+            return self.tesseract_det_inference(arrays)
```

Comment:
A bug: It uses Tesseract for detection tasks whenever `model == 'Tesseract'`, which still holds true when we want to perform recognition task with Tesseract. 
```bash
python mmocr/utils/ocr.py --det None --recog Tesseract --print-result.
```